test/unit/math/rev/fun/lbeta_test.cpp

```diff
@@ -8,6 +10,101 @@
 #include <cmath>
 #include <functional>
 
+namespace lbeta_test_internal {
+// TODO(martinmodrak) the function here should be replaced by helpers for
+// testing identities once those are available
+
+struct identity_tolerances {
+  stan::test::relative_tolerance value;
+  stan::test::relative_tolerance gradient;
+};
+
+template <class F1, class F2>
+void expect_identity(const std::string& msg,
```

Comment:
What's the identity being tested?

Reply:
This is just a stub that would be replaced by tools developed in #1671 .

Reply:
But added the tested identities as comments.